numba/tests/npyufunc/test_cuda_gufunc.py

```diff
@@ -0,0 +1,190 @@
+from __future__ import print_function, absolute_import
```

Comment:
There are certainly multiple axes to organize them. I have grouped them together as ufunc tests.